sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveStrategies.scala

```diff
@@ -213,7 +213,9 @@ case class RelationConversions(
 
   private def isConvertible(storage: CatalogStorageFormat): Boolean = {
     val serde = storage.serde.getOrElse("").toLowerCase(Locale.ROOT)
-    serde.contains("parquet") && conf.getConf(HiveUtils.CONVERT_METASTORE_PARQUET) ||
+    val inputFormat = storage.inputFormat.getOrElse("").toLowerCase(Locale.ROOT)
+    serde.contains("ParquetHiveSerDe") && inputFormat.contains("MapredParquetInputFormat") &&
```

Comment:
It seems like you are conducting a comparison that is case-sensitive, even though you have already converted the text to lowercase.

Reply:
Absolutely right :)